### ISSUE TITLE
Implement new interface ``\Darling\RoadyRoutes\interfaces\collections\PositionNameCollection`:

### ISSUE THREAD

**Issue by sevidmusic** [sevidmusic/RoadyRoutes#33]
Implement new interface ``\Darling\RoadyRoutes\interfaces\collections\PositionNameCollection`:

```

<?php

namespace Darling\RoadyRoutes\interfaces\collections;

use \Darling\RoadyPositionNames\interfaces\identifiers\PositionName;

/**
 * A PositionNameCollection defines a collection of PositionNames.
 *
 */
interface PositionNameCollection
{

    /**
     * Return a numerically indexed array of PositionNames.
     *
     * @return array<int, PositionName>
     *
     */
    public function collection(): array;

}

```

**Comment by sevidmusic** [comment]
Resolved iin 5a32873cb56c1bf49a530df6fea2833411811ddf

**Comment by sevidmusic** [comment]
Resolved in 5a32873cb56c1bf49a530df6fea2833411811ddf
